Tools/CISupport/build-webkit-org/steps.py

```diff
@@ -501,7 +501,7 @@ class CompileLLINTCLoop(CompileWebKit):
 
 class CompileJSCOnly32(CompileWebKit):
     name = 'compile-jsc-32bit'
-    build_command = ["linux32", "perl", "Tools/Scripts/build-jsc", "--32-bit", "--cmakeargs=\"-DUSE_LIBBACKTRACE=OFF -DDEVELOPER_MODE=ON -DENABLE_OFFLINE_ASM_ALT_ENTRY=1 -DCMAKE_CXX_FLAGS='-fuse-ld=gold -Wl,--no-map-whole-files -Wl,--no-keep-memory -Wl,--no-keep-files-mapped -Wl,--no-mmap-output-file -fno-omit-frame-pointer' -DCMAKE_C_FLAGS='-fuse-ld=gold -Wl,--no-map-whole-files -Wl,--no-keep-memory -Wl,--no-keep-files-mapped -Wl,--no-mmap-output-file -fno-omit-frame-pointer' -DUSE_LD_LLD=OFF\""]
+    build_command = ["linux32", "perl", "Tools/Scripts/build-jsc", "--32-bit", "--cmakeargs", "-DUSE_LIBBACKTRACE=OFF -DDEVELOPER_MODE=ON -DENABLE_OFFLINE_ASM_ALT_ENTRY=1 -DCMAKE_CXX_FLAGS='-fuse-ld=gold -Wl,--no-map-whole-files -Wl,--no-keep-memory -Wl,--no-keep-files-mapped -Wl,--no-mmap-output-file -fno-omit-frame-pointer' -DCMAKE_C_FLAGS='-fuse-ld=gold -Wl,--no-map-whole-files -Wl,--no-keep-memory -Wl,--no-keep-files-mapped -Wl,--no-mmap-output-file -fno-omit-frame-pointer' -DUSE_LD_LLD=OFF"]
```

Comment:
Another way of passing arguments is to add them into the 'additionalArguments' parameter in `config.json`, for example: https://github.com/WebKit/WebKit/blob/main/Tools/CISupport/ews-build/config.json#L404

I would leave things as they're now, but you may want to try that approach in case this fix doesn't work (for some reason).

Also, when passing arguments to the `build-webkit` command is possible to pass the 'cmakeargs' argument several times. So it would be possible to break down CMake arguments into several parameters. For instance:

"cmakeargs", "foo", "cmakeargs", "bar", "cmakeargs", "quax"...

Reply:
@justinmichaud I'm not a reviewer, so you'd need a thumbs up review from an actual reviewer.